### PR TITLE
[Swift] Release Swift 5.2.1

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 7867dcd..6a7c6b8 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -35,6 +35,12 @@ swift:5.2-bionic-slim
 swift:5.2-slim
 swift:5.2-xenial
 swift:5.2-xenial-slim
+swift:5.2.1
+swift:5.2.1-bionic
+swift:5.2.1-bionic-slim
+swift:5.2.1-slim
+swift:5.2.1-xenial
+swift:5.2.1-xenial-sim
 swift:bionic
 swift:bionic-slim
 swift:latest
diff --git a/swift_latest/Dockerfile b/swift_latest/Dockerfile
index e2bb2de..235f876 100644
--- a/swift_latest/Dockerfile
+++ b/swift_latest/Dockerfile
@@ -27,8 +27,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu18.04
-ARG SWIFT_BRANCH=swift-5.2-release
-ARG SWIFT_VERSION=swift-5.2-RELEASE
+ARG SWIFT_BRANCH=swift-5.2.1-release
+ARG SWIFT_VERSION=swift-5.2.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_slim/Dockerfile b/swift_slim/Dockerfile
index 55dc783..ba6e5e7 100644
--- a/swift_slim/Dockerfile
+++ b/swift_slim/Dockerfile
@@ -17,8 +17,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu18.04
-ARG SWIFT_BRANCH=swift-5.2-release
-ARG SWIFT_VERSION=swift-5.2-RELEASE
+ARG SWIFT_BRANCH=swift-5.2.1-release
+ARG SWIFT_VERSION=swift-5.2.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_xenial-slim/Dockerfile b/swift_xenial-slim/Dockerfile
index 5d55926..fefc1ae 100644
--- a/swift_xenial-slim/Dockerfile
+++ b/swift_xenial-slim/Dockerfile
@@ -17,8 +17,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu16.04
-ARG SWIFT_BRANCH=swift-5.2-release
-ARG SWIFT_VERSION=swift-5.2-RELEASE
+ARG SWIFT_BRANCH=swift-5.2.1-release
+ARG SWIFT_VERSION=swift-5.2.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
diff --git a/swift_xenial/Dockerfile b/swift_xenial/Dockerfile
index 9e8f395..d8ebd50 100644
--- a/swift_xenial/Dockerfile
+++ b/swift_xenial/Dockerfile
@@ -27,8 +27,8 @@ RUN export DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true && ap
 # uid                  Swift 5.x Release Signing Key <swift-infrastructure@swift.org
 ARG SWIFT_SIGNING_KEY=A62AE125BBBFBB96A6E042EC925CC1CCED3D1561
 ARG SWIFT_PLATFORM=ubuntu16.04
-ARG SWIFT_BRANCH=swift-5.2-release
-ARG SWIFT_VERSION=swift-5.2-RELEASE
+ARG SWIFT_BRANCH=swift-5.2.1-release
+ARG SWIFT_VERSION=swift-5.2.1-RELEASE
 ARG SWIFT_WEBROOT=https://swift.org/builds/
 
 ENV SWIFT_SIGNING_KEY=$SWIFT_SIGNING_KEY \
```

</details>

**Comment by yosifkit** [comment]
Travis really is green: https://travis-ci.org/github/docker-library/official-images/builds/669044967

File: library/swift [hunk 1]
```diff
@@ -6,20 +6,20 @@ Maintainers: Ted Kremenek <kremenek@apple.com> (@tkremenek),
...
 
-Tags: 5.2-slim, 5.2-bionic-slim, bionic-slim, slim
-GitCommit: 6b02e77e0be14e9dfa42a2965db8be91a7ffd918
+Tags: 5.2.1-slim, 5.2-slim, 5.2.1-bionic-slim, 5.2-bionic-slim, bionic-slim, slim
+GitCommit: 4aa206c264d5792491e8679fe96ca6f4754a9c87
 Directory: 5.2/ubuntu/18.04/slim
 
-Tags: 5.2-xenial-slim, xenial-slim
-GitCommit: 6b02e77e0be14e9dfa42a2965db8be91a7ffd918
+Tags: 5.2.1-xenial-sim, 5.2-xenial-slim, xenial-slim
```
Comment (yosifkit) [inline library/swift]: `sim` -> `slim` :wink:

Comment (shahmishal) [inline library/swift]:
```suggestion
Tags: 5.2.1-xenial-slim, 5.2-xenial-slim, xenial-slim
```

Comment (shahmishal) [reply library/swift]: Fixed, thanks!

**Comment by yosifkit** [comment]
(diff unchanged from above)
Build test of #7715; 95edbe99; `amd64` (`swift`):

```console

$ bashbrew build swift:5.2.1
Using bashbrew/cache:e4a25dccc3a1db8c14640baedd7077a24e751655479608c52389e739921c164b (swift:5.2.1)
Tagging swift:5.2.1
Tagging swift:5.2
Tagging swift:5.2.1-bionic
Tagging swift:5.2-bionic
Tagging swift:bionic
Tagging swift:latest

$ test/run.sh swift:5.2.1
testing swift:5.2.1
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.2.1-xenial
Using bashbrew/cache:dfd17a512b83e51fcceca9bc5fa850602df2df1d311141914221411875f7ef7b (swift:5.2.1-xenial)
Tagging swift:5.2.1-xenial
Tagging swift:5.2-xenial
Tagging swift:xenial

$ test/run.sh swift:5.2.1-xenial
testing swift:5.2.1-xenial
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.2.1-slim
Using bashbrew/cache:8e708436095e04ecb6d4eef0724f0f5ae729e5708da47317e9087857a37c1de3 (swift:5.2.1-slim)
Tagging swift:5.2.1-slim
Tagging swift:5.2-slim
Tagging swift:5.2.1-bionic-slim
Tagging swift:5.2-bionic-slim
Tagging swift:bionic-slim
Tagging swift:slim

$ test/run.sh swift:5.2.1-slim
testing swift:5.2.1-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.2.1-xenial-slim
Using bashbrew/cache:9989f10e932bf469af2dbdac23b32e6cd1fe3a5a044e13a4781e645c63e933bc (swift:5.2.1-xenial-slim)
Tagging swift:5.2.1-xenial-slim
Tagging swift:5.2-xenial-slim
Tagging swift:xenial-slim

$ test/run.sh swift:5.2.1-xenial-slim
testing swift:5.2.1-xenial-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.1.5
Using bashbrew/cache:327271eecb893bd250f3a698054e0c1c825ca1a482c4171b262d476809f9347f (swift:5.1.5)
Tagging swift:5.1.5
Tagging swift:5.1
Tagging swift:5.1.5-bionic
Tagging swift:5.1-bionic

$ test/run.sh swift:5.1.5
testing swift:5.1.5
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.1.5-xenial
Using bashbrew/cache:e9274c7f0bb673daccf601dd34a660f2d1cd863fdf595da7c96cac5390e0fc42 (swift:5.1.5-xenial)
Tagging swift:5.1.5-xenial
Tagging swift:5.1-xenial

$ test/run.sh swift:5.1.5-xenial
testing swift:5.1.5-xenial
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.1.5-slim
Using bashbrew/cache:c276575b1f7fb022710f5cd829cbab928cf063ae504ee3b67cd69db6b9a8d291 (swift:5.1.5-slim)
Tagging swift:5.1.5-slim
Tagging swift:5.1-slim
Tagging swift:5.1.5-bionic-sim
Tagging swift:5.1-bionic-slim

$ test/run.sh swift:5.1.5-slim
testing swift:5.1.5-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.1.5-xenial-slim
Using bashbrew/cache:9c4aeecda97b5a4501b10f770161fd8d193991018891dfc3419f157d09187078 (swift:5.1.5-xenial-slim)
Tagging swift:5.1.5-xenial-slim
Tagging swift:5.1-xenial-slim

$ test/run.sh swift:5.1.5-xenial-slim
testing swift:5.1.5-xenial-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.0.3
Using bashbrew/cache:08aa80e798e86c24a30112a16e9da8f7ee7f7ed6f6066baec707d7a2913e7e30 (swift:5.0.3)
Tagging swift:5.0.3
Tagging swift:5.0
Tagging swift:5.0.3-bionic
Tagging swift:5.0-bionic

$ test/run.sh swift:5.0.3
testing swift:5.0.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.0.3-xenial
Using bashbrew/cache:060fbecd3dc9426cc035d28ef697a471e8d6a865e1de8c29a1f5218723230248 (swift:5.0.3-xenial)
Tagging swift:5.0.3-xenial
Tagging swift:5.0-xenial

$ test/run.sh swift:5.0.3-xenial
testing swift:5.0.3-xenial
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:5.0.3-slim
Using bashbrew/cache:b9fbf001681ddbc3d5763006ad97a3ae784fe274ce462fac24bea65986a537aa (swift:5.0.3-slim)
Tagging swift:5.0.3-slim
Tagging swift:5.0-slim
Tagging swift:5.0.3-bionic-slim
Tagging swift:5.0-bionic-slim

$ test/run.sh swift:5.0.3-slim
testing swift:5.0.3-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:5.0.3-xenial-slim
Using bashbrew/cache:b2b8b799487fc4b0893b87c4cd20ced5585cc604c7e9eba069ec30ca63acfd2a (swift:5.0.3-xenial-slim)
Tagging swift:5.0.3-xenial-slim
Tagging swift:5.0-xenial-slim

$ test/run.sh swift:5.0.3-xenial-slim
testing swift:5.0.3-xenial-slim
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build swift:4.2.4
Using bashbrew/cache:0f316ef8ea0765c903ef98827d83d567ae880235870939bdb60b68a29ef55234 (swift:4.2.4)
Tagging swift:4.2.4
Tagging swift:4.2
Tagging swift:4

$ test/run.sh swift:4.2.4
testing swift:4.2.4
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:4.1.3
Using bashbrew/cache:16cbb0c67a4dfb8f65d366cfcf0c78b0c9cf71901429f2bf3926f3c81a132e04 (swift:4.1.3)
Tagging swift:4.1.3
Tagging swift:4.1

$ test/run.sh swift:4.1.3
testing swift:4.1.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed


$ bashbrew build swift:4.0.3
Using bashbrew/cache:a83248a8b41f0d4ac5b6b02d1a3c6d1ff2b21b64a7aaaafba73742b84e176f39 (swift:4.0.3)
Tagging swift:4.0.3
Tagging swift:4.0

$ test/run.sh swift:4.0.3
testing swift:4.0.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'swift-hello-world' [5/5]...passed

```

**Pull request merged**